src/DetailsView/components/issue-filing-dialog.tsx

```diff
@@ -104,28 +104,29 @@ export class IssueFilingDialog extends React.Component<IssueFilingDialogProps, I
     }
 
     private onPrimaryButtonClick = (ev: React.SyntheticEvent<Element, Event>) => {
-        const newData = this.state.selectedBugFilingService.getSettingsFromStoreData(this.state.bugServicePropertiesMap);
-        const service = this.state.selectedBugFilingService.key;
+        const newData = this.state.selectedIssueFilingService.getSettingsFromStoreData(this.state.issueServicePropertiesMap);
+        const service = this.state.selectedIssueFilingService.key;
         this.props.deps.userConfigMessageCreator.saveIssueFilingSettings(service, newData);
-        this.props.bugFileTelemetryCallback(ev);
+        this.props.issueFileTelemetryCallback(ev);
```

Comment:
Not sure if this is the right time/place but I'd prefer fileIssueTelemetryCallback